### PR TITLE
update admin mail setup for sieve

### LINKED ISSUE #66: Remove procedures/mail_setup
This seems more of a friendly instruction than Redbrick Infrastructure maintenance guide. Perhaps we should remove it from the docs and put it into more appropriate place (wiki, with link in the onboarding doc?)

### PR DESCRIPTION
**Pull request by butlerx**
Closes #66

### CODE REVIEW

File: docs/procedures/committeechangeover.md [hunk 1]
```diff
@@ -96,7 +86,7 @@ $ ldapmodify -D cn=root,ou=ldap,o=redbrick -x -y /etc/ldap.secret
 
 Add the webmaster to the webmaster mailing list
 (webmaster@lists.redbrick.dcu.ie) on
-[lists](http://lists.redbrick.dcu.ie/mailman/admin/webmaster/)
+[lists](https://lists.redbrick.dcu.ie/mailman/admin/webmaster/)
```
Comment (m1cr0man) [inline docs/procedures/committeechangeover.md]: Can you change this to here: https://lists.redbrick.dcu.ie/postorius/lists/webmaster.lists.redbrick.dcu.ie/

File: docs/procedures/mail_setup.md [hunk 2]
```diff
@@ -1,142 +1,109 @@
-# Mail Setup
+# Admin Mail Setup
```
Comment (m1cr0man) [inline docs/procedures/mail_setup.md:1]: Is there need for this? We have this doc'd on the wiki (https://wiki.redbrick.dcu.ie/index.php/Mutt). I'm not a fan of duplication.

**Review by m1cr0man — CHANGES_REQUESTED** [review]
Can you rebase onto master also?

File: docs/procedures/mail_setup.md (hunk 2 above)
Comment (butlerx) [reply docs/procedures/mail_setup.md:1]: the labeling was the main thing

Comment (m1cr0man) [reply docs/procedures/mail_setup.md:1]: Fair enough.

**Comment by butlerx** [comment]
rebased

**Review by m1cr0man — APPROVED** [review]

**Pull request merged**